src/spacewar/core.cljc

```diff
@@ -66,8 +67,11 @@
 (defn setup []
   (let [vmargin 30
         hmargin 5
-        world (if (.exists (io/file "starwars.world"))
-                (read-string (slurp "starwars.world"))
```

Comment:
In the ClojureScript port, we use `edn/read-string` for both Clojure and ClojureScript. This is uniform, and actually more secure.